openage/convert/main.py

```diff
@@ -175,8 +180,14 @@ def main(args, error):
     from ..assets import get_asset_path
     outdir = get_asset_path(args.output_dir)
 
+    # mount the config folder at "cfg/"
```

Comment:
Umm not sure I did it right here. 

There's also `interactive_browser` function at line 175 which requires the `args` to have the `GameEdition` and `GameExpansion` objects to be available. I should push it after `convert_assets` function right?

Reply:
No, the interactive browser is a separate module that can be invoked before conversion. I would recommend you also pass it the cfg dir and replicate the game edition/expansion reading and subsequent version detection to the interactive browser.